spec/unit/worker_spec.rb

```diff
@@ -165,12 +165,6 @@ def around_perform(job)
         worker.send(:log, :warn, 'my-message')
         expect(logger_io.string).to match(/my-message/)
       end
-
-      it 'can save log trace' do
```

Comment:
The function itself gone. It will reappear in qless customers (wherever needed) with appropriate tests.